src/styles/button.scss

```diff
@@ -0,0 +1,24 @@
+.to-buy {
+  display: block;
+  width: 166px;
+  margin-left: $padding-left;
+  border: 1px solid $blue_color;
+  margin-top: 16px;
+  background-color: $blue_color;
```

Comment:
The variable `$blue_color` is used but not defined. Please define this variable to avoid compilation errors.